xplat/web/restart.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
So this works, and I'll merge it if you don't feel like working on this any more, but I have a couple suggestions if you're game:
- You don't need to kill the server, you can just rerun the emcc command in build.sh to regenerate the wasm package
- Actually, you don't even need to rerun emcc, you can just run the [file packager manually](https://emscripten.org/docs/porting/files/packaging_files.html#packaging-using-the-file-packager-tool) -- since we are "preloading" rather than "embedding" the files, assets are requested over the network on page load anyway